src/ui/Templates/TemplateFromAScriptTag.ts

```diff
@@ -83,17 +83,17 @@ export class TemplateFromAScriptTag {
 
   parseFieldsAttributes(): IFieldsToMatch[] {
     const dataSet = this.scriptTag.dataset;
-    return _.chain(dataSet)
+    return chain(dataSet)
       .map((value, key: string) => {
-        const match = key.match(/field([a-z0-9]*)/i);
+        const match = key.match(/field([a-zA-Z0-9_\.]*)/i);
```

Comment:
Should this change be documented somewhere?

Reply:
No, it was a bug.